src/libs/actions/Task.ts

```diff
@@ -735,7 +736,7 @@ function clearOutTaskInfoAndNavigate(reportID?: string, chatReport?: OnyxEntry<O
     }
     if (accountID > 0) {
         const accountLogin = allPersonalDetails?.[accountID]?.login ?? '';
-        setAssigneeValue(accountLogin, accountID, reportID, chatReport);
+        setAssigneeValue(accountLogin, accountID, reportID, chatReport, skipConfirmation);
```

Comment:
I think `skipConfirmation` param is passed as `isCurrentUser` to `setAssigneeValue`. And maybe `skipConfirmation` can even be removed if `skipShareDestination` is not required per the earlier comment. Please check.

Reply:
Ah, good catch. Fixed.